Tests/IntegrationTests.swift

```diff
@@ -35,17 +41,36 @@ final class IntegrationTests: XCTestCase {
     }
   }
 
-  func testSendXRP() {
+  func testGetTransactionStatus() {
+    do {
+      let transactionStatus = try client.getTransactionStatus(for: .successfulTransactionHash)
+      XCTAssertEqual(transactionStatus, .succeeded)
+    } catch {
+      XCTFail("Failed retrieving transaction hash with error: \(error)")
+    }
+  }
+
+  // MARK: - Legacy Protocol Buffers
+
+  func testGetBalance_legacy() {
+    do {
+      _ = try legacyClient.getBalance(for: Wallet.testWallet.address)
+    } catch {
+      XCTFail("Failed retrieving balance with error: \(error)")
+    }
+  }
+
+  func testSendXRP_legacy() {
```

Comment:
is there a send for the new client?

Reply:
Implemented in PR #99, which has these commits in it.